### DISCUSSION TITLE
ESM-IF1 weights not downloading automatically

### DISCUSSION THREAD

**Discussion by Oran-G** [facebookresearch/esm#223] [General]
How can I download weights for ESM-IF! using the ESM library? when I run sem.pretrained for the correct model, I am told that the model is initializing randomly and will not predict correct sequences. how can I get the correct weights?

**Comment by tomsercu** [comment]
It may be a warning related to `contact_prediction_head` - can you copy paste the exact error message? See #170 outstanding todo

**Reply by Oran-G** [reply]
less of an error and more of a warning --  did not quit the script

> /ext3/miniconda3/lib/python3.9/site-packages/fair_esm-0.4.2-py3.9.egg/esm/pretrained.py:174: UserWarning: Regression weights not found, predicting contacts will not produce correct results.
initialized
It is the same error message as found in #170
